### PR TITLE
Repo setup

### PR DESCRIPTION
**Pull request by SchrodingersStruggle**
# Initial setup for repository
  - Added a devcontainer, updated requirements.txt, and added linting to repository.

## Contents
  - .devcontainer/python/devcontainer.json - Devcontainer to set up python in repository.
  -  .github/workflows/pylinttest.yml - Allows linting in PR's
  - requirements.txt - Edited existing file, added numpy, pandas, matplotlib, and pylint.

## Label:
 - Added organization label to characterize this pull request.

## Reviewer
  - I would suggest @laserlab to review this.

### CODE REVIEW

File: .github/workflows/pylinttest.yml
Comment (laserlab) [inline .github/workflows/pylinttest.yml:1]: In the last Homework we had that separated. You can talk to @kylemasc917 and @HeshamElsaman if that was more convenient. I would merge it like this for now though.

**Review by laserlab — APPROVED** [review]
Looks good,

check for my comment about splitting the linting / testing

**Comment by SchrodingersStruggle** [comment]
@kylemasc917 If I were to separate the linting and pytest, would I need to clear the cache of pylint like in flake 8, or would that not be necessary?

**Comment by kylemasc917** [comment]
Not that I know of, but you'll have to test it out and see.

File: .github/workflows/pylinttest.yml
Comment (HeshamElsaman) [reply .github/workflows/pylinttest.yml:1]: Not separating them would be better, not 100% sure, though.
I'd suggest not separating them and adding comments to the .yaml file..

**Comment by SchrodingersStruggle** [comment]
Okay I will keep them merged for now, as suggested by @laserlab, and merge this request so people can work with linting in the repository.

**Pull request merged**